pkg/api/testing/fuzzer.go

```diff
@@ -117,8 +117,9 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			c.FuzzNoCustom(j) // fuzz self without calling this function again
 		},
 		func(j *api.DaemonControllerStatus, c fuzz.Continue) {
```

Comment:
You should be able to omit this function entierly-- the fuzzer should just work?